default.nix

```diff
@@ -1,16 +1,85 @@
-with (import (fetchTarball https://github.com/NixOS/nixpkgs/archive/fb235c98d839ae37a639695ad088d19ef8382608.tar.gz) {});
-# NOTE: when bumping nixpkgs, also update nixpkgs-src.json and .travis.yml
+{ cluster ? "mainnet" }:
+let
+  pkgs = import (import ./fetchNixpkgs.nix (builtins.fromJSON (builtins.readFile ./nixpkgs-src.json))) { config = {}; overlays = []; };
+  packages = self: {
+    inherit pkgs;
+    master_config = {
+      daedalus_darwin_url = "https://update-cardano-mainnet.iohk.io/Daedalus-installer-1.1.0.408.pkg";
+      cardano_rev = "eef095";
+      daedalus_hash = "1b0iy6c5nlsa2n2ylcc2kfm8ykkmdp6ncnx4lwwvc6f0662cf175";
```

Comment:
This should default to parsing .git?

Reply:
the cardano rev controls which cardano it fetches, so there is no `.git` before that
and the daedalus_darwin_url is to fetch the pre-npm'd javascript, mayflower will be replacing that